2006Scape Client/src/main/java/Game.java

```diff
@@ -12636,6 +12683,9 @@ public void keyPressed(KeyEvent keyevent)
 				}
 
 		}
+		  if (keyevent.getKeyCode() == KeyEvent.VK_PRINTSCREEN && keyevent.isControlDown()) {
```

Comment:
i know its only screenshots but we should put this behind a QoL toggle

Reply:
I'd prefer it be it's own flag then like --enable-screenshots, separate from the other QoL toggles, since it's an important enough feature it should stand on it's own imo.

Reply:
yeah thats okay, also add a toggle like you did for zooming please :pray:

Reply:
Thanks, it's pushed now :)